qiskit_neko/__init__.py

```diff
@@ -13,4 +13,4 @@
 """qiskit-neko integration test suite."""
 
 
-from qiskit_neko.version import __version__
+from qiskit_neko.version import __version__ as __version__
```

Comment:
?

Reply:
This was the only ruff error after moving off of pylint (which was causing the install issues). It said either do this or add `__all__` and use `__version__`.

Reply:
that's a pretty stupid lint rule, but ok haha